mock.py

```diff
@@ -0,0 +1,126 @@
+# /********************************************************************************
```

Comment:
should this license header fit to "our" velocitas header?

Reply:
Done